apiBase/APIRequestUtils.es6.js

```diff
@@ -105,8 +105,10 @@ export const validateData = (data, method, apiName, validator) => {
 export const runForm = (apiOptions, method, path, data, parseBody, parseMeta) => {
   if (!(apiOptions && method && path && data)) { throw new NoModelError(); }
 
+  const type = method === 'patch' ? 'json' : 'form';
```

Comment:
I can move it into another method. At first I didn't want to as I was hoping we'd get to the request'ing refactor sooner, but seeing it here makes me want to just move it.